src/e2e-test/features/spanner/source/SpannerToSpanner_Additional.feature

```diff
@@ -0,0 +1,118 @@
+@Spanner_Source @SPANNER_TEST
+Feature: Spanner source - Verification of Additional Spanner to Spanner successful data transfer without using connections
+
+  @SPANNER_TEST @EXISTING_SPANNER_SINK
+  Scenario: To verify data is getting transferred from Spanner source to existing Spanner sink without using connection functionality
```

Comment:
Change the scenario name and provide more clear statement. 
_To verify data is getting transferred from Spanner source to Spanner sink having existing table name without using connection functionality_

Reply:
Noted